primitives/src/policy.rs

```diff
@@ -156,6 +156,37 @@ impl Policy {
     /// 25 MB.
     pub const HISTORY_CHUNKS_MAX_SIZE: u64 = 25 * 1024 * 1024;
 
+    /// Percentage of minimum supporting stake to perform upgrade.
+    /// A value of `80` means 80% of the stake needs to support the upgrade.
+    pub const UPGRADE_MIN_SUPPORT: u8 = 80;
+
+    /// This function is used to determine if a validator signalled for a specific upgrade.
+    /// For now, this is checking the first two bytes of the signal data.
```

Comment:
```suggestion
    /// This is checking the first two bytes of the signal data.
```

"For now" suggests to me that we're currently already planning something else.